### ISSUE TITLE
crc32 and film conflict with anime fansub releases

### ISSUE THREAD

**Issue by zodman** [guessit-io/guessit#245]
the guessit think the F are for film on F9967949 

```
For: [PuyaSubs!] Digimon Adventure tri - 01 [720p][F9967949].mkv
GuessIt found: {
    "film_title": "PuyaSubs! Digimon Adventure tri", 
    "episode": 1, 
    "screen_size": "720p", 
    "crc32": "F9967949", 
    "film": 9967949, 
    "container": "mkv", 
    "mimetype": "video/x-matroska", 
    "type": "episode"
}

**Comment by Toilal** [comment]
Thanks for reporting, I'll fix this before 2.0 final release.

**Comment by zodman** [comment]
guessit version from master branch
